src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -3844,6 +3848,7 @@ private BoundExpression BindNullCoalescingOperator(BinaryExpressionSyntax node,
                     else
                     {
                         ReportDiagnosticsIfObsolete(diagnostics, leftConversion, node, hasBaseReceiver: false);
+                        CheckConstraintLanguageVersionAndRuntimeSupportForConversion(node, leftConversion, diagnostics);
```

Comment:
I didn't see usage of null coalescing operators in `StaticAbstractMembersInInterfacesTests.cs`. Similar to the foreach element scenario, should we have a test which ensures we report appropriate LangVersion and runtime support errors on these null coalescing operand conversions?

Reply:
> I didn't see usage of null coalescing operators in StaticAbstractMembersInInterfacesTests.cs. Similar to the foreach element scenario, should we have a test which ensures we report appropriate LangVersion and runtime support errors on these null coalescing operand conversions?

Tracked by the same issue as the other.